babycenter/templates/core/section_page.html

```diff
@@ -33,13 +33,34 @@ <h3 class="article-list__title article-list__title{{self.get_effective_extra_sty
   {% if articles %}
   <div class="step-links l-indent l-indent--tall">
     {% if articles_paginated.has_previous %}
-        <a href="?p={{ articles_paginated.previous_page_number }}">&larr;</a>
+      <a href="?p={{ articles_paginated.previous_page_number }}">< Prev</a>
     {% endif %}
-    <span class="current">
-        Page {{ articles_paginated.number }} of {{ articles_paginated.paginator.num_pages }}
-    </span>
+
+      {% get_pages articles_paginated as pages %}
```

Comment:
I think this needs to start with `with` and have a closing `endwith` at the end.